### PR TITLE
Add OwnerEmail and change some text in sc-ec2-windows-jumpcloud

### PR DESCRIPTION
**Pull request by tthyer**
Minor windows fixes

### CODE REVIEW

File: ec2/sc-ec2-windows-jumpcloud.yaml [hunk 1]
```diff
@@ -32,20 +35,25 @@ Parameters:
...
-    Type: AWS::SSM::Parameter::Value<AWS::EC2::Image::Id>
+    Type: AWS::EC2::Image::Id
     Default: ami-074545a6fb2313e2c
   VolumeSize:
     Description: The EC2 volume size (in GB)
     Type: Number
     Default: 30
     MinValue: 10
     MaxValue: 2000
+  OwnerEmail:
```
Comment (zaro0508) [inline ec2/sc-ec2-windows-jumpcloud.yaml]: isn't this already a required tag in the SC?  would SC end user need to set this twice?

Comment (tthyer) [reply ec2/sc-ec2-windows-jumpcloud.yaml]: That's right...but it's needed for 01_associate_jc_systems. Also doing it that way (requiring twice) in the linux template. I could rewrite these to pull OwnerEmail from tags as with TagRootVolume though.

Comment (zaro0508) [reply ec2/sc-ec2-windows-jumpcloud.yaml]: only setting it once would be desirable but that can be in a follow on PR if you want

Comment (zaro0508) [reply ec2/sc-ec2-windows-jumpcloud.yaml]: only setting it once would be desirable but that can be in a follow on PR if you want

Comment (tthyer) [reply ec2/sc-ec2-windows-jumpcloud.yaml]: Agree will work on that next.

**Pull request merged**